frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -16,27 +18,25 @@ class PopupCreateFile extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
-      appName: 'file', // must remain 'file' because it is the name of the react built app (which contains File and PopupCreateFile)
+      appName: 'file', // INFO - CH - 2018-08-28 - must remain 'file' because it is the name of the react built app (which contains File and PopupCreateFile)
       config: props.data ? props.data.config : debug.config,
       loggedUser: props.data ? props.data.loggedUser : debug.loggedUser,
       workspaceId: props.data ? props.data.workspaceId : debug.workspaceId,
       folderId: props.data ? props.data.folderId : debug.folderId,
-      uploadFile: null,
+      uploadFileList: [],
```

Comment:
if `uploadFileList` is supposed to store the list of file about to be uploaded, rename it accordingly. It's name wasn't great but now there is `uploadedFileList` it's even worse